### ISSUE TITLE
New topics 

### ISSUE THREAD

**Issue by kmonterr** [18F/nsf-sbir#1374]
Hi Sarah!

We'll need to update the topics and the topic PDF on the website. We don't want it going live until March 4, but there's plenty to update! Can you start working on a test branch with all of the changes?

We'll also need to revisit our API conversation...@Benschrag

Here's who is covering what -

•	Henry Ahn, Biomedical (BM) Technologies, telephone: 703-292-7069, email: hahn@nsf.gov
•	Peter Atherton, Information Technologies (IT), Artificial Intelligence (AI), Quantum Information Technologies (QT), telephone: 703-292-8772, email:patherto@nsf.gov
•	Anna Brady-Estevez, Chemical Technologies (CT), Distributed Ledger (DL),  Environmental Technologies (ET), telephone: 703-292-7077, email: abrady@nsf.gov
•	Nancy U. Kamei, Digital Health (DH) and Medical Devices (MD), telephone: 703-292-7236, email:nkamei@nsf.gov
•	Steven Konsek, Advanced Materials (AM), , Semiconductors (S), telephone: 703-292-7021, email:skonsek@nsf.gov
•	Rajesh Mehta, Educational Technologies and Applications (EA), telephone: 703-292-2174, email:rmehta@nsf.gov
•	Muralidharan S. Nair, Energy and Power Systems (EP), Robotics (R), Sensors (SE), Wireless Technologies (W), telephone: 703-292-7059, email: mnair@nsf.gov
•	Linda K. Molnar, Advanced Manufacturing (M) and Nanotechnology (MN), telephone: 703-292-8316, email: lmolnar@nsf.gov
•	Ben Schrag, Other Topics (OT), telephone: 703-292-8323, email: bschrag@nsf.gov
•	Rick Schwerdtfeger, Instrumentation and Hardware Systems (IH), Internet of Things (I), Photonics (PH), Space (SP) telephone: 703-292-8353, email: rschwerd@nsf.gov
•	Ruth M. Shuman, Biological Technologies (BT), telephone: 703-292-2160, email:rshuman@nsf.gov

Thank you!!

**Comment by seakhtar** [comment]
Sure @kmonterr , I'll get a new branch ready. Thanks for the PD and topic info, that'll help get us started!

**Comment by kmonterr** [comment]
Let me know if you need anything with this!

**Comment by seakhtar** [comment]
Sure! I've been discussing the plan of attack regarding how to handle the new topics and the API issue with the rest of the team. 
Once the topics PDF becomes available, I'll be able to do the bulk of the updates-- I think I usually use it to make all of the topics pages (since it has all the subtopics and descriptions).

**Comment by kmonterr** [comment]
[SBIR Topic updates combined_2.12.2019final.pdf](https://github.com/18F/nsf-sbir/files/2886319/SBIR.Topic.updates.combined_2.12.2019final.pdf)
oh no! Here it is!! Let me know if you want the word version!

**Comment by seakhtar** [comment]
Thanks Kelly! If you do have the word version handy, that one might go a little faster, but this works too if not.

**Comment by kmonterr** [comment]
[SBIR Topic updates combined_2.12.2019final.docx](https://github.com/18F/nsf-sbir/files/2886692/SBIR.Topic.updates.combined_2.12.2019final.docx)
Here you go!

**Comment by seakhtar** [comment]
Perfect- thank you! I'll have the test branch up soon--

**Comment by seakhtar** [comment]
Hi @kmonterr -

Here is the test branch I have for the new topics:
https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/topicsUpdate0219/portfolio/

I'm still combing through to make sure I got everything, but wanted to give you a chance to start looking it over as well. Let me know if you notice anything off!

I had just a couple quick notes:
-On the topics document, for advanced materials AM6 and AM5 seem to be out of order. I wanted to check if that was intentional or not? Once I know for sure I'll get the pdf uploaded to the test branch as well.
-Each topic page has it own header image, so for the new ones I've just used the same images as some of the other pages. They might not match up correctly though, so let me know if you'd like me to change them around or remove them altogether.

**Comment by benschrag** [comment]
Hi all,

Looks good!  The only thing I noted to fix is to please make sure that the name of the CT topic is just “Chemical Technologies”.

The AM topic is a typo – you can switch the 5 and 6 and leave the rest as is.  Thanks for catching that!

Cheers,
Ben

**Comment by kmonterr** [comment]
[SBIR Topic updates combined_2.21.2019final.pdf](https://github.com/18F/nsf-sbir/files/2894363/SBIR.Topic.updates.combined_2.21.2019final.pdf)

Thank you, Sarah! I will go through the test branch this morning. I'm sure Ben would catch anything, but just in case I'll check too.

Here's the updated PDF. Thanks for catching that subtopic code!

Best,
Kelly

**Comment by kmonterr** [comment]
We'll get you all of the feedback by Wednesday, Feb. 27!

Here's another fix - Under (EW5), please delete “Human” before “Other Robotics Technologies”

**Comment by kmonterr** [comment]
Under Biological Technologies topic, someone has added a new topic – Other Biotechnology Technologies (BT13).  I suggest that this be changed to Other Biological Technologies.

Here's the PDF with the fixed language.
[SBIR Topic updates combined_2.22.2019final.pdf](https://github.com/18F/nsf-sbir/files/2894710/SBIR.Topic.updates.combined_2.22.2019final.pdf)

**Comment by kmonterr** [comment]
Please add "Bay Labs" and "GrokStyle" as featured companies in the AI topic area. We might have a few more coming too.

**Comment by kmonterr** [comment]
Hi Sarah! Here's an updated PDF and word doc. We changed the IT topic and subtopics and the AI topic and subtopic order. Thank you!

[SBIR Topic updates combined_2.25.2019final.pdf](https://github.com/18F/nsf-sbir/files/2901420/SBIR.Topic.updates.combined_2.25.2019final.pdf)
[SBIR Topic updates combined_2.25.2019final.docx](https://github.com/18F/nsf-sbir/files/2901421/SBIR.Topic.updates.combined_2.25.2019final.docx)

**Comment by seakhtar** [comment]
Thanks @kmonterr! I'll get it uploaded and work on getting all of these last changes into place.

**Comment by kmonterr** [comment]
Thank you @seakhtar !

**Comment by kmonterr** [comment]
Hi @seakhtar - Do you mind adding "Bay Labs" and "GrokStyle" as featured companies in the AI topic area?

Thank you!

**Comment by seakhtar** [comment]
Yes! Sorry, didn't realize those weren't showing up yet. I'll get it fixed.

**Comment by kmonterr** [comment]
👍
